src/DependenciesPackager/DependenciesPackager.csproj

```diff
@@ -18,4 +18,17 @@
     <PackageReference Include="System.Diagnostics.Process" Version="4.1.0" />
   </ItemGroup>
 
+  <!-- packaging settings -->
+  <PropertyGroup>
+    <NoPackageAnalysis>true</NoPackageAnalysis>
+    <NuspecFile>$(MSBuildThisFileDirectory)$(MSBuildProjectName).nuspec</NuspecFile>
+  </PropertyGroup>
+
+  <Target Name="SetPackageDependencies" BeforeTargets="GenerateNuspec" DependsOnTargets="Publish">
+    <PropertyGroup>
```

Comment:
That may be true when developing MSBuild targets in an SDK, but seems like overkill for this one target in this one project.

Also, FWIW the SDK team is slowly trying to move off *DependsOn properties. These are the legacy of the first MSBuild which didn't have AfterTargets/BeforeTargets.